network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_authors.html

```diff
@@ -10,7 +10,9 @@
     {% spaceless %}
     {% if show_full_info %}{% trans "By" %} {% endif %}
     {% for author in authors %}
-    <span>{% if authors_length > 1 and forloop.last %}{% trans "and" %} {% endif %}<b class="tw-font-extrabold">{{ author.name }}{% if not forloop.last %}{% if forloop.counter != authors_length|add:-1%},{% endif %} {% endif %}</b></span>
+        <a href="{% routablepageurl page=blog_index url_name='blog-author-detail' profile_slug=author.slug  %}">
+            <span>{% if authors_length > 1 and forloop.last %}{% trans "and" %} {% endif %}<b class="tw-font-extrabold">{{ author.name }}{% if not forloop.last %}{% if forloop.counter != authors_length|add:-1%},{% endif %} {% endif %}</b></span>
+        </a>
```

Comment:
**Suggestion** Sorry, I missed this before. This currently will make the author texts blue and on the last name also have the link on the "and". I think it might be better to wrap only `{{ author.name }}` in the link and also add `class="tw-text-black"` to the anchor tag.

Reply:
Thanks :) fixed with 626e6c8adc402eee1a2ae92420f915d3070b9563